lib/utils/report.mjs

```diff
@@ -213,27 +218,94 @@ function isDisabled(ruleName, startLine, disabledRanges) {
 	return false;
 }
 
-/** @param {Problem & { line: number }} problem */
+/**
+ * @param {Problem & { line: number }} problem
+ * @returns {boolean}
+ */
 function isFixApplied({ fix, line, result: { stylelint }, ruleName }) {
+	if (!fix) return false;
+
 	const { disabledRanges, config = {}, fixersData } = stylelint;
 
-	if (!isFn(fix)) {
-		return false;
-	}
+	if (!config.fix) return false;
+
+	if (isFixDisabled(line, ruleName, config, disabledRanges)) return false;
```

Comment:
suggestion

If you want you can reduce the number of arguments passed to `isFixDisabled`.
Marginally better? Not sure.

```suggestion
	const { config, fixersData } = stylelint;

	if (!config?.fix) return false;

	if (isFixDisabled(line, ruleName, stylelint)) return false;
```
 and inside `isFixDisabled`
```js
	const { config = {}, disabledRanges } =  stylelint;
```

Reply:
I'd rather keep it as is.